imap-proto/src/parser/rfc3501.rs

```diff
@@ -505,7 +505,8 @@ named!(response_data<Response>, do_parse!(
         mailbox_data |
         message_data_expunge |
         message_data_fetch |
-        resp_capability
+        resp_capability |
+        metadata_solicited
```

Comment:
This doesn't compile for me: "cannot find function `metadata_solicited` in this scope"